python/pyspark/sql/connect/functions.py

```diff
@@ -2373,6 +2374,109 @@ def hours(col: "ColumnOrName") -> Column:
 
 hours.__doc__ = pysparkfuncs.hours.__doc__
 
+
+def make_dt_interval(
```

Comment:
Scala side code is fine to me, need someone familiar with python for further review